packages/frontend/src/renderer/store/reducers.ts

```diff
@@ -13,16 +17,18 @@ import { socketReducer } from '../sagas/socket/socket.slice'
 import { modalsReducer } from '../sagas/modals/modals.slice'
 
 import appHandlers from './handlers/app'
-import waggleHandlers from './handlers/waggle'
-import contactsHandlers from './handlers/contacts'
-import channelHandlers from './handlers/channel'
-import directMessages from './handlers/directMessages'
-import directMessageChannelHandlers from './handlers/directMessageChannel'
-import mentionsHandlers from './handlers/mentions'
-import criticalErrorHandlers from './handlers/criticalError'
-import whitelistHandlers from './handlers/whitelist'
 
-const reduxStorage = createElectronStorage()
+const dataPath = process.env.APPDATA || (process.platform === 'darwin' ? process.env.HOME + '/Library/Application Support' : process.env.HOME + '/.config')
```

Comment:
Why do you need to compose path for macos manually like that? What about Windows?

Reply:
It's for all 3 systems

Reply:
It's because with new version we have to specify the appData path for redux persist, and it is not accessible in renderer, hence I compose it manually